pkg/exec/exec.go

```diff
@@ -64,3 +64,13 @@ func (exe *Executor) GoInstall(ctx context.Context, path, gobin string) (int, er
 	cmd.Env = append(os.Environ(), "GOBIN="+gobin)
 	return exe.exec(ctx, cmd)
 }
+
+func (exe *Executor) HdiutilDetach(ctx context.Context, mountPath string) (int, error) {
+	cmd := exe.command(exec.Command("hdiutil", "detach", mountPath, "-quiet"))
+	return exe.exec(ctx, cmd)
+}
+
+func (exe *Executor) HdiutilAttach(ctx context.Context, dmgPath, mountPoint string) (int, error) {
+	cmd := exe.command(exec.Command("hdiutil", "attach", dmgPath, "-mountpoint", mountPoint, "-quiet"))
```

Comment:
I'm considering if we should set `-quiet` option or not.

`-quiet` option removes the error message.

```console
$ hdiutil attach  get.volta.sh
hdiutil: attach failed - image not recognized

$ hdiutil attach  get.volta.sh -quiet
$ echo $?
1
```

On the other hand, `-quiet` option makes the log clean.

Reply:
How about removing the quiet option when AQUA_LOG_LEVEL=debug?

Reply:
Interesting.
Or it is also good to output logs only when the command failed.

Reply:
I don't think you can tell the success or failure of hdiutil at the stage of attaching the -quiet option.  If I run without the -quiet option, I feel that the hdiutil log should have flowed to the terminal without permission.  So I think that there is no choice but to control with the presence or absence of the log level in advance.  How about that?

Reply:
Done. cd205a7faa0d21026ae03bbb45e5750c9e5af129

Reply:
> How about removing the quiet option when AQUA_LOG_LEVEL=debug?

This idea is interesting, but I think if `hdiutil attach` command fails the error log should be always outputted.
Otherwise, users can't understand why `hdiutil attach` failed.

Reply:
> I don't think you can tell the success or failure of hdiutil at the stage of attaching the -quiet option. If I run without the -quiet option, I feel that the hdiutil log should have flowed to the terminal without permission. So I think that there is no choice but to control with the presence or absence of the log level in advance.

Sorry. I missed your comment.

I think we can judge if the command succeeded or failed as the following before outputting the log.
What do you think?

https://github.com/aquaproj/aqua/blob/cd205a7faa0d21026ae03bbb45e5750c9e5af129/pkg/exec/exec.go#L47-L58